userspace/libpman/include/libpman.h

```diff
@@ -293,7 +294,7 @@ extern "C"
 	 * @brief Enable BPF-capture if we have previously
 	 * disabled it.
 	 */
-	int pman_enable_capture(bool *tp_set);
+	int pman_enable_capture(bool *ppm_sc_set, bool *ppm_tp_set);
```

Comment:
Question: why separating the two sets? From the loop below, my naive understanding is that we could attain this by passing only the ppm_sc_set.

Reply:
Probably yep; i think we can merge the 2 sets into a single param.